experiments/generic/pod_cpu_hog/pod_cpu_hog_k8s_job.yml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: pod-cpu-hog- 
+spec:
+  template:
+    metadata:
+      labels:
+        experiment: pod-cpu-hog
+    spec:
+      # Placeholder that is updated by the executor for automated runs
+      # Provide appropriate SA (with desired permissions) if executed manually
+      serviceAccountName: nginx 
```

Comment:
Can we provide %CHAOS_SERVICE_ACCOUNT% here

Reply:
done!